src/components/pagination/pagination.tsx

```diff
@@ -142,7 +152,11 @@ export const EuiPagination: FunctionComponent<Props> = ({
 
     if (compressed) {
       centerPageCount = (
-        <EuiText size="s" className="euiPagination__compressedText">
+        <EuiText
+          size="s"
+          css={paginationStyles.euiPagination__compressedText}
+          data-test-subj="euiPagination__compressedText"
```

Comment:
Wait, why add this `data-test-subj`? And why add it in CSS casing instead of JS camel casing (`euiPaginationCompressedText`)?

Reply:
Or wait, shouldn't this have just been left in as a regular className?
```suggestion
          className="euiPagination__compressedText"
```